.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@ repos:
         pass_filenames: true
         require_serial: false
 -   repo: https://github.com/python/black
-    rev: 22.1.0
+    rev: 22.3.0
```

Comment:
also need to update environment.yml and requirements-dev.txt

Reply:
good catch - might need a CI check for this else we'll never remember each time 😄